gulpfile.js

```diff
@@ -142,4 +153,34 @@ gulp.task('watch', function() {
 	gulp.watch('**/*.css', ['lint:css']);
 });
 
+// Server code from http://blog.overzealous.com/post/74121048393/why-you-shouldnt-create-a-gulp-plugin-or-how-to
+gulp.task('server', ['build:extensions'], function(callback) {
+	var log = gutil.log;
+	var colors = gutil.colors;
+
+	var devApp = connect();
+	devApp.use(connectLogger('dev'));
+	devApp.use(connectStatic('.'));
+
+	// Automatically rebuild Extensions on script changes
+	gulp.watch('**/*.js', ['build:extensions']);
+	gulp.watch('**/*.css', ['build:extensions']);
+
+	var devServer = http.createServer(devApp).listen(31337);
+
+	devServer.on('error', function(error) {
+		log(colors.underline(colors.red('ERROR'))+' Unable to start server!');
+		callback(error); // we couldn't start the server, so report it and quit gulp
+	});
+
+	devServer.on('listening', function() {
+		var devAddress = devServer.address();
+		var devHost = devAddress.address === '0.0.0.0' ? 'localhost' : devAddress.address;
+		var url = 'http://' + devHost + ':' + devAddress.port;
+
+		log('Started dev server at ' + colors.magenta(url));
```

Comment:
This log doesn't seem to give a valid IP address for me.  I'm seeing:
```sh
Started dev server at http://:::31337
```

`devAddress` doesn't seem to resolve to an actual IP:
```sh
{ address: '::', family: 'IPv6', port: 31337 }
```

Is that just me?